# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/events/EventListenerManager.cpp
# Commit: ea775927f4ab
# Full Hash: ea775927f4ab98bb722ed493061868ad917af829
# Author: Yulia Startsev <ystartsev@mozilla.com>
# Date: 2022-02-18 21:52:29
# Regressor Bug: 1742437
# File Overlap Count: 1
# Description:
#   Bug 1742437 - move mWebExtGlobal from ScriptLoadRequest to DOMScriptLoadContext; r=jonco,smaug
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D136744
# ==============================================================================

diff -r c33b42cabd55 -r ea775927f4ab dom/events/EventListenerManager.cpp
--- a/dom/events/EventListenerManager.cpp	Fri Feb 18 12:45:49 2022 +0000
+++ b/dom/events/EventListenerManager.cpp	Fri Feb 18 12:45:49 2022 +0000
@@ -1236,7 +1236,7 @@
 
   RefPtr<ScriptFetchOptions> fetchOptions = new ScriptFetchOptions(
       CORS_NONE, aElement->OwnerDoc()->GetReferrerPolicy(),
-      aElement->OwnerDoc()->NodePrincipal(), nullptr);
+      aElement->OwnerDoc()->NodePrincipal());
 
   RefPtr<EventScript> eventScript =
       new EventScript(fetchOptions, uri, aElement);